src/components/ConfigValidation/ConfigIndicator.tsx

```diff
@@ -11,34 +14,38 @@ interface Props {
   size?: string;
 }
 
-interface Validation {
```

Comment:
Since we're exporting this now, can we name it something else, like `ValidationResult`?